mqtt/mqtt-generic/src/settings.rs

```diff
@@ -176,6 +176,7 @@ mod tests {
                     RetainedMessagesConfig::new(1000, Duration::from_secs(60 * DAYS)),
                     SessionConfig::new(
                         Duration::from_secs(60 * DAYS),
+                        Duration::from_secs(DAYS), // 1d
```

Comment:
```suggestion
                        Duration::from_secs(1*DAYS), 
```
?

Reply:
there are more places with similar suggestion

Reply:
Clippy doesn't like that.